recmetrics/metrics.py

```diff
@@ -139,7 +139,69 @@ def _ark(actual: list, predicted: list, k=10) -> int:
 
     return score / len(actual)
 
-def mark(actual: List[list], predicted: List[list], k=10) -> int:
+def _pk(actual: list, predicted: list, k) -> float:
```

Comment:
Question: Let me know if this implementation is too different than the  `_ark` / `mark` one -- I figured it made sense to just factor this out. But in broader terms, maybe it makes sense to use the `recommender_precision` method inside of `_apk`?

Reply:
Update: I removed `_pk` in favor of just using the pre-existing precision calc. hopefully that refactor is okay!

Reply:
This is great! thanks for this improvement!